File: BarCode SDK/C#/QR Code with Sms Message/QrCodeSmsTemplate.cs
//*******************************************************************************************//
//                                                                                           //
// Get Your API Key: https://app.pdf.co/signup                                               //
// API Documentation: https://developer.pdf.co/api/index.html                                //
//                                                                                           //
// Note: Replace placeholder values in the code with your API Key                            //
// and file paths (if applicable)                                                            //
//                                                                                           //
//*******************************************************************************************//


using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CreateBarCode
{
    class QrCodeSmsTemplate
    {
        #region Constructors

        public QrCodeSmsTemplate() { }
        public QrCodeSmsTemplate(string Number, string Message)
        {
            this.Number = Number;
            this.Message = Message;
        }

        #endregion

        #region Properties

        public string Number { get; set; }
        public string Message { get; set; }

        #endregion

        #region Overloaded Methods

        public override string ToString()
        {
            if (string.IsNullOrEmpty(Number) && string.IsNullOrEmpty(Message))
                return base.ToString();

            return $"smsto:{Number}:{Message}";
        }

        #endregion
    }
}

File: BarCode SDK/VB.NET/QR Code With Sms Message/QrCodeSmsTemplate.vb
'*******************************************************************************************'
'                                                                                           '
' Get API Key https://app.pdf.co/signup                                                     '
' API Documentation: https://developer.pdf.co/api/index.html                                '
'                                                                                           '
' Note: Replace placeholder values in the code with your API Key                            '
' and file paths (if applicable)                                                            '
'                                                                      '
'*******************************************************************************************'


Public Class QrCodeSmsTemplate

#Region "Constructors"

    Public Sub New()
    End Sub

    Public Sub New(ByVal Number As String, ByVal Message As String)
        Me.Number = Number
        Me.Message = Message
    End Sub

#End Region


#Region "Properties"

    Public Property Number As String
    Public Property Message As String

#End Region

#Region "Overloaded Methods"

    Public Overrides Function ToString() As String

        If String.IsNullOrEmpty(Number) AndAlso String.IsNullOrEmpty(Message) Then Return MyBase.ToString()

        Return $"smsto:{Number}:{Message}"

    End Function

#End Region


End Class
